hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/AccessHeatMapEndpoint.java

```diff
@@ -88,4 +92,19 @@ public Response getReadAccessMetaData(
     }
     return Response.ok(entityReadAccessHeatMapResponse).build();
   }
+
+  private static void checkIfHeatMapFeatureIsEnabled() {
+    FeatureProvider.Feature heatMapFeature = null;
+    List<FeatureProvider.Feature> allDisabledFeatures =
+        FeatureProvider.getAllDisabledFeatures();
+    for (FeatureProvider.Feature feature : allDisabledFeatures) {
+      if (feature.getFeatureName().equals("HeatMap")) {
```

Comment:
nit: to avoid NPE.
Constant should be on left side e.g. `"HeatMap".equals(feature.getFeatureName())` or use `Objects.equals()`.

Reply:
> nit: to avoid NPE. Constant should be on left side e.g. `"HeatMap".equals(feature.getFeatureName())` or use `Objects.equals()`.

Thanks @hemantk-12  for noticing this, however this line will not hit NPE because if any disabled features are there, then this list will be populated  as Feature enum with non-null feature name, however , yes, as a good practice, we should use Constant on left side. Thank you.